File: app/src/main/java/com/ronaker/app/utils/Debug.java
package com.ronaker.app.utils;

import android.util.Log;

public class Debug {

    public static void Log(String TAG, String Message) {
        Log.d(TAG, Message);
    }

    public static void Log(String TAG, Exception ex) {
        Log(TAG, "Exception ", ex);

    }

    public static void Log(String TAG, String Message, Exception ex) {
        if (ex != null)
            Message = Message + " --> " + ex.getMessage();

        Log.e(TAG, Message);
        if (ex != null)
            ex.printStackTrace();
    }

}

File: app/src/main/java/com/ronaker/app/utils/Debug.kt
package com.ronaker.app.utils;

import android.util.Log

object Debug {

    fun Log(TAG: String, Message: String?) {
        if (Message != null)
            Log.d(TAG, Message)

    }

    fun Log(TAG: String, ex: Exception?) {
        if (ex != null)
            Log(TAG, "Exception ", ex)

    }

    fun Log(TAG: String, Message: String?, ex: Exception?) {

        var message1 = Message?:""
        if (ex != null)
            message1 = message1 + " --> " + ex.message

        Log.e(TAG, message1)
        ex?.printStackTrace()
    }

}
